src/components/icons/apple-icon/apple-icon.stories.tsx

```diff
@@ -3,7 +3,7 @@ import React from "react";
 import { AppleIcon } from "./apple-icon";
 
 export default {
-  title: "icons/social/Apple Icon",
+  title: "icons/Social/Apple Icon",
```

Comment:
Why is it in uppercase again ?

Reply:
Well, this looks like a path, but this is used for this part:
<img width="202" alt="image" src="https://user-images.githubusercontent.com/31956107/105147768-e8936200-5b01-11eb-822b-1bc80d6001bb.png">

the first one is full upper cased, but the second is not, so for consistency, I changed it.